packages/network/test/unit/agent_spec.ts

```diff
@@ -31,7 +32,7 @@ chai.use(sinonChai)
 
 const PROXY_PORT = 31000
 const HTTP_PORT = 31080
-const HTTPS_PORT = 31443
+const HTTPS_PORT = 443
```

Comment:
can you think of a way to make this less confusing for devs if they run this spec as non-root? otherwise tests will just fail with "ELISTEN can't listen on 127.0.0.1:443"

could check if `process.geteuid() !== 0` and log a special warning, for example

Reply:
[`f81145c` (#20107)](https://github.com/cypress-io/cypress/pull/20107/commits/f81145c0d9c91e4f26f1d16cfce5e5997053db21)

[`0900d42` (#20107)](https://github.com/cypress-io/cypress/pull/20107/commits/0900d42e9d7e4da2f134a4b7f03b21a44676370f)